resources/skins/nudus/html_wrapper.html

```diff
@@ -0,0 +1,237 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!DOCTYPE html>
+<html lang="en">
+<!--
+  ___________________________
+< I'm an expert in my field. >
+  ---------------------------
+         \   ^__^
+          \  (oo)\_______
+             (__)\       )\/\
+                 ||----w |
+                 ||     ||
+-->
+<head>
+    <style>
+    :root {
+        --base-unit: 10px;
+        /* fusion border grey */
+        --grey1: #B8B8B8;
+
+        /* some grey */
+        --grey2: #CFCFCF;
+        --grey10: #343434;
+        --grey20: #F1F1F1;
+    }
+    body, h1, h2, h3, h4, h5, h6,
+    p, blockquote, pre, hr,
+    dl, dd, ol, ul, figure {
+        margin: 0;
+        padding: 0;
+    }
+    h1, h2, h3, h4, h5, h6,
+    p, blockquote, pre,
+    ul, ol, dl, figure {
+        margin-bottom: var(--base-unit);
+    }
+    body {
+        cursor: default;
+        font-kerning: normal;
+        -webkit-text-size-adjust: 100%;
+    }
+    pre,
+    code {
+        cursor: text;
+    }
+    h1, h2, h3, h4, h5, h6 {
+        font-weight: 600 !important;
+    }
+    h1 { font-size: 1.25em  !important; }

```

Comment:
Does your skin work fine when changing font size for article preview in settings?

Reply:
Perfectly fine.